api/proto/teleport/legacy/types/types.proto

```diff
@@ -4591,6 +4591,8 @@ message WindowsDesktopFilter {
   string HostID = 1 [(gogoproto.jsontag) = "host_id"];
   // Name is the name of the desktop.
   string Name = 2 [(gogoproto.jsontag) = "name"];
+
+  repeated bool NonAD = 3 [(gogoproto.jsontag) = "non_ad"];
```

Comment:
Why is this a `repeated bool`?

Reply:
It was removed in current implementation.